src/components/main-view/main-view.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import axios from 'axios';
+
+import { MovieCard } from '../movie-card/movie-card';
+import { MovieView } from '../movie-view/movie-view';
+
+export class MainView extends React.Component {
+	constructor() {
+		super();
+
+		this.state = {
+			movies: null,
+			selectedMovie: null,
+		};
+	}
+
+	componentDidMount() {
+		axios
+			.get('<https://myflix-movies-app.herokuapp.com/movies')
```

Comment:
Have a look at this url, if you copy the entire string in the `axios.get()` function and paste it in your browser what happens?

Change the string to: `'https://myflix-movies-app.herokuapp.com/movies'`

There is another issue at the moment where your API is returning an empty array but we'll get to that one